addons/mail/static/src/models/thread/thread.js

```diff
@@ -1161,6 +1161,22 @@ function factory(dependencies) {
             return [['replace', allAttachments]];
         }
 
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeAttachmentsCountLabel() {
+            return _.str.sprintf(this.env._t("%s Attachment(s)"), this.allAttachments.length);
```

Comment:
This is kind of bad in terms of translation in some languages, if we want to have it nicely it would be better to have conditions and different strings at least for 0, 1 and plural.

Also is there any reason to capitalize the word?

Reply:
I seem to recall that some languages such as Arabic require a dual form, that is, a special plural form to describe pairs of two things. You may want to consider this special case in addition to those already pointed out by Seb.

Reply:
Removed, used the same approach as in the top bar, added aria label to the image. So, neither extra translations nor computed fields are required anymore.